pkg/controller/health_check.go

```diff
@@ -56,7 +56,7 @@ func (hc *HealthCheck) GetHandler(ctx *gin.Context) {
 	}
 	err := json.NewEncoder(ctx.Writer).Encode(healthInfo)
 	if err != nil {
-		hc.logger.Println("error writing response body", err.Error())
+		log.Info(ctx, fmt.Sprintf("error writing response body: %s", err.Error()))
```

Comment:
I'm going to stop reviewing all the logger usage here. Can you please verify that you use the logger properly? I see a lot of misuse here. Info instead of errors. Wrong params, etc.

Reply:
@alexeykazakov i used info for places that we used Println. I assumed we would have used error otherwise.. but this can be easily changed...

Reply:
fixed as part of: 00bf17d961e783ddd6ad166872fdd5903f82a7c1